Apsitvarkom.Models/Enumerations.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Apsitvarkom.Models;
+
+/// <summary>
+/// Class for storing all enumerated types for models.
+/// </summary>
+public class Enumerations
+{
+    /// <summary>Enumeration type for pollution levels of the location.</summary>
+    public enum LocationSeverityLevel
+    {
+        None = 0,
```

Comment:
Do we need a `None` severity? not sure how this would be useful

Reply:
Not sure - perhaps could be used for closed records, but it's up for discussions, really.